index.html

```diff
@@ -0,0 +1,349 @@
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <footer>
+      <div class="main-contact">
+        <div class="left-section">
+          <h3 class="footer-title">
+            Please get in touch with me incase you would like me to contribute
+            to your project
+          </h3>
+        </div>
+        <div class="right-section">
+          <form action="https://formspree.io/f/mleajywy" method="post">
+            <div class="name-section">
+              <input
+                type="text"
+                id="firstname"
+                name="Firstname"
+                placeholder="First Name"
+                maxlength="30"
+              />
+              <input
+                type="text"
+                id="lastname"
+                name="Lastname"
+                placeholder="Last Name"
+                maxlength="30"
+              />
+            </div>
+            <div class="email-section">
+              <input
+                type="text"
+                id="email"
+                name="email"
+                placeholder="Email Address"
+              />
+            </div>
+            <textarea
+              id="message"
+              name="message"
+              placeholder="Its a pleasure seeing you !"
+              maxlength="500"
+            ></textarea>
+            <input
+              type="submit"
+              value="Get in Touch"
+              class="button-link link-btn"
+            />
+          </form>
```

Comment:
Your form needs to be validated according to requirements. e.g, a user should not be able to submit the form if one or more of the fields is empty. Please read the form instructions go and implement that in your project.